pallets/accelerated-bn-crypto/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "accelerated-bn-crypto"
+version = "0.1.0"
+description = "A pallet showing usage of the native implementation of BN254 primitives backed by Substrate host functions"
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license = "Apache-2.0"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+ark-bn254 = { version = "0.4.0", default-features = false, features = ["curve"] }
+ark-bn254-ext = { git = "https://github.com/DimitrisPapac/accelerated-bn-cryptography.git", default-features = false }
+ark-ec = { version = "0.4.0", default-features = false }
+ark-ff = { version = "0.4.0", default-features = false }
+ark-serialize = { version = "0.4.0", default-features = false }
+ark-scale = { features = ["hazmat"], version = "0.0.12", default-features = false }
+ark-std = { version = "0.4.0", default-features = false }
+codec = { workspace = true }
+native = { path = "../../native", default-features = false, features = ["bn254"] }
+frame-benchmarking = { workspace = true }
+frame-support = { workspace = true }
+frame-system = { workspace = true }
+sp-core = { workspace = true }
+sp-io = { workspace = true }
+sp-runtime = { workspace = true }
+scale-info = { workspace = true, features = ["derive"] }
+
+[dev-dependencies]
+pallet-balances = { workspace = true, features = ["std"] }
```

Comment:
Unused `pallet-balances` dependency, once we use the default mock runtime configuration